.github/workflows/edFi.loadtools manual.yml

```diff
@@ -0,0 +1,63 @@
+name: EdFi.LoadTools Manually triggered build
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+
+env:
+  INFORMATIONAL_VERSION: "5.4"
+  CONFIGURATION: "Release"
+  AZURE_ARTIFACT_URL: "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
+  AZURE_ARTIFACT_NUGET_KEY: ${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}
+  VSS_NUGET_EXTERNAL_FEED_ENDPOINTS : '{"endpointCredentials": [{"endpoint": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json","password": "${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}"}]}'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
```

Comment:
Please change to version hash instead of `@v1`

Reply:
I forgot what the issue was before with why I did not do this, but now I made the change to see it again. With a hash for this it shows a warning in the build output (but will still complete the run) as you can see here: https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/runs/6159523661?check_suite_focus=true#step:3:1

This same warning does not come up on the checkout action though when using a hash, and I found nothing mentioned in the docs around the Github action for why the warning would be coming up.

Related, I also noticed now that the default workflow template references setup-dotnet@v2, so should I go ahead and move this over to the hash for v2?

Reply:
Perhaps you grabbed the hash value for 1.0.0, whereas @v1 would run the latest @v1.y.z?

Yes, please try out v2.